spec/tests/yaml/swiftnav/sbp/test_observation.yaml

```diff
@@ -349,305 +349,5 @@ tests:
     payload: KKNEGC4HEIRr2XkOlP//ASyvkwCBQsh9lB3+/5kpDrEDgLL4iI9C/v/eEriFDZ41soYqGwIA7R4hYBYmU1Z2qG///y040Fge
     preamble: '0x55'
     sender: '0x4c3'
```

Comment:
oh weird.  the log I used to generate the unit test must have had only one type of ephemeris message in it.  I'll regenerate the unit tests with a log with both types of ephemeris.  Thanks for pointing that out.